### ISSUE TITLE
ERROR: CMD ended with status code 1

### ISSUE THREAD

**Issue by vini8cs** [bcgsc/transabyss#31]
Hi! I'm encountering an issue with Trans-Abyss that appears to be a problem in the code. When executing the abyss-filtergraph command, it seems that too many arguments are being passed to the function abyss-filtergraph:

`CMD: bash -euo pipefail -c 'abyss-filtergraph --shim --assemble --kmer=25 --island=0 --remove=/home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.braid.cids --graph=/home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.filtered.adj1 /home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-1.adj /home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-1.fa > /home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.path'
`
`abyss-filtergraph: too many arguments`
`Try abyss-filtergraph --help for more information.`
`ERROR: CMD ended with status code 1`

I attempted to address this issue by removing the last argument, achieved by deleting this line, but the problem persists:

`remove_rref_cmd_params1.append(last_good_fasta)`

While attempting to merge the contigs, another issue arises:

`
CMD: bash -euo pipefail -c 'abyss-filtergraph --shim --assemble --kmer=25 --island=0 --remove=/home/apis/Gabriela_data_metagenomics/SRR1370499
2/25_assembly_trans_abyss-unitigs.r1.braid.cids --graph=/home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.f
iltered.adj1 /home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-1.adj > /home/apis/Gabriela_data_metagenomics/SRR137049
92/25_assembly_trans_abyss-unitigs.r1.path'
`
`Elapsed time: 0 h 0 m 17 s`
`CMD: bash -euo pipefail -c 'MergeContigs --kmer=25 --out=/home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.
filtered.fa --adj --graph=/home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.filtered.adj /home/apis/Gabriel
a_data_metagenomics/SRR13704992/25_assembly_trans_abyss-1.fa /home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs
.r1.filtered.adj1 /home/apis/Gabriela_data_metagenomics/SRR13704992/25_assembly_trans_abyss-unitigs.r1.path'` 
`error: unexpected ID: 113'`
`Aborted (core dumped)`
`Elapsed time: 0 h 0 m 47 s`
`ERROR: CMD ended with status code 134`

What should I do?

**Comment by kmnip** [comment]
Please confirm whether you have the correct version of ABySS installed.
As mentioned in our [README](https://github.com/bcgsc/transabyss), Trans-ABySS only works with ABySS version 2.0.x, e.g. https://github.com/bcgsc/abyss/releases/tag/2.0.3
If you installed Trans-ABySS and its dependencies with conda, you shouldn't have any issues.

**Comment by vini8cs** [comment]
However, for some reason, I am unable to download the 2.0.3 version using conda, even when I specify the version. The installation only proceeds with the 1.5.2 version. I will attempt to install it from the source instead.

**Comment by vlad0x00** [comment]
That often happens if there are conflicting versions with other installed packages. You can also try installing abyss and transabyss in a fresh conda environment.

**Comment by kmnip** [comment]
Yes, I agree with @vlad0x00. Please install Trans-ABySS into a dedicated conda environment, e.g.
```
conda create -n transabyss -c bioconda transabyss
```
Activate this environment and try your assembly again:
```
conda activate transabyss
transabys ...
```
